CHANGELOG.md

```diff
@@ -1,3 +1,48 @@
+## [5.0.0](https://github.com/commercetools/merchant-center-application-kit/compare/v4.4.0...v5.0.0) (2019-01-03)
+
+## BREAKING CHANGES 💣
+
+When using the `@commercetools-frontend/permissions` package, we no longer rely on a static list of permission scopes (e.g. `[permissions.ViewProducts]`), instead we expect the list of `demandedPermissions` to be a **String** (e.g. `['ViewProducts']`). This is necessary as we need more flexibility in defining and using new permissions.
```

Comment:
@tdeekens @qmateub please check if that's correct and/or if I missed something.

Reply:
Looks correct to me 👍

Reply:
It is correct. Just maybe a bit strangely phrased. TL;DR

1. Instead of hard-coding supported permissions (by constants from the prrmissions package) we now validate required permissions to actually be configured to gain more flexibility
2. The legacy (and long time deprecated) format of permissions with mode and resource is no longer supported

The change is two fold. The fact that we do not validate/require any passed permissions to be part of the constants is not a breaking change (likely) as consumers of app-kit do not set this up and the mc-be still exposes the same permissions. The removal of the legacy format however is. Even though it was introduced before app-kit was made public.

Reply:
Thanks, I'll update the text